src/models/DockerComposatorComponent.js

```diff
@@ -0,0 +1,31 @@
+import { Component } from 'leto-modelizer-plugin-core';
+
+/**
+ * Specific Docker compose component.
+ */
+class DockerComposatorComponent extends Component {
```

Comment:
Prefere to have one commit by model, in each commit you will have model and according unit tests